thoth/storages/sync.py

```diff
@@ -98,8 +99,11 @@ def sync_solver_documents(
     graceful: bool = False,
     graph: Optional[GraphDatabase] = None,
     is_local: bool = False,
+    randomize_listing: Optional[bool] = False,
```

Comment:
Can we have this configurable via environment variables on a library level? That way we can control this behavior directly from deployment manifests.